scripts/trim-low-abund.py

```diff
@@ -100,19 +106,20 @@ def main():
     ###
```

Comment:
On Wed, Feb 11, 2015 at 08:06:56AM -0800, Michael R. Crusoe wrote:
> > @@ -100,19 +106,20 @@ def main():
> >      ###
> 
> Instead of quitting we should let the user know that there were duplicate filenames and they were ignored.

I disagree; this could be part of the -f flag, though.